portfolio/src/main/webapp/about.html

```diff
@@ -1 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Annie+Use+Your+Telescope|Nanum+Pen+Script&display=swap" rel="stylesheet">
+  </head>
+  <body>
+  <div class="nav">
+    <a class="active" href="index.html">Home</a>
+    <a href="aboutme.html">Translate</a>
+    <a href="about.html">About Me</a>
+  </div>
+    <div id="content">
+    <div style="background-color:white;background-image:url(//www.html.am/images/backgrounds/background-image-2.gif);background-repeat:repeat;"></div>
```

Comment:
Ideally keep styles in CSS. See this part of the style guide for some more info, https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns.